docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3"
+
+services:
+  chatgpt-api-server:
+    build: .
+    ports:
+      - "8080:8080"
+    command: ["ChatGPT-API-server", "8180", "<api-key>"]
```

Comment:
This looks a bit off. Is that the right port number?

Reply:
line 8 has typo?